x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_integration_event_filters_card.test.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import uuid from 'uuid';
+import { waitFor } from '@testing-library/react';
+import * as reactTestingLibrary from '@testing-library/react';
+import { createAppRootMockRenderer } from '../../../../../../../common/mock/endpoint';
+import { EventFiltersHttpService } from '../../../../../event_filters/service';
+import { FleetIntegrationEventFiltersCard } from './fleet_integration_event_filters_card';
+
+jest.mock('../../../../../event_filters/service');
+
+const EventFiltersHttpServiceMock = EventFiltersHttpService as jest.Mock;
```

Comment:
Suggestion: use the eventFiltersListQueryHttpMock instead

Reply:
done 43675eaf3b09597f0bd1301c5fc0579e79dcff4d